Dockerfile.dev

```diff
@@ -9,10 +9,10 @@ RUN yum install -y libvirt-devel && yum clean all && rm -rf /var/cache/yum/*
 
 
 ADD bin/hiveadmission /opt/services/
-ADD bin/hive-operator /opt/services/
+ADD bin/operator /opt/services/hive-operator
 ADD bin/manager /opt/services/
 ADD bin/hiveutil /usr/bin/
-ADD bin/hive-apiserver /opt/services
+#ADD bin/hive-apiserver /opt/services
```

Comment:
stray commented line?

Reply:
Also something that needs to be restored.